### PR TITLE
Encoded postfixes fix

### PR DESCRIPTION
**Pull request by Connor-Colenso**

### CODE REVIEW

File: src/main/java/appeng/util/ReadableNumberConverter.java [hunk 1]
```diff
@@ -23,7 +23,7 @@ public enum ReadableNumberConverter implements ISlimReadableNumberConverter, IWi
     /**
      * String representation of the sorted postfixes
      */
-    private static final char[] ENCODED_POSTFIXES = "KMGTPE".toCharArray();
+    private static final char[] ENCODED_POSTFIXES = "KMBTQU".toCharArray();
```
Comment (boubou19) [inline src/main/java/appeng/util/ReadableNumberConverter.java:26]: You can't just get rid of the metric prefixes like that

**Review by boubou19 — CHANGES_REQUESTED** [review]

File: src/main/java/appeng/util/ReadableNumberConverter.java (hunk 1 above)
Comment (S4mpsa) [reply src/main/java/appeng/util/ReadableNumberConverter.java:26]: Agreed, this is a travesty!

**Review by Glease — CHANGES_REQUESTED** [review]
do this in AE2 colen edition. changes like this should not be merged in mainline distribution

**Comment by Glease** [comment]
Or, at the very least, a config option.

**Pull request closed without merging**